NSS-Day-Cohort-49/react-nutshell-state-slingers
# Lam messages

#### Changes Made
1. Added file `MessageProvider.js, MessageList.js, MessageCard.js, and Message.css` to `messages` directory.
1. Modified file `MessageList.js and MessageCard.js along with NavBar.js` to include `display of message lists and functionality to add and delete a message, and to display the active link` .
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout lam-messages
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user clicks on the messages navigation, the messages list should display, when user types a message in message input and clicks on button to send, the new message should render at bottom of page and when user clicks on delete button, the message should delete.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
